shell/platform/darwin/macos/framework/Source/FlutterViewControllerTestUtils.mm

```diff
@@ -27,4 +27,4 @@ id CreateMockViewController(NSString* pasteboardString) {
   }
 }
 
-}
+}  // namespace flutter::testing
```

Comment:
`google-readability-namespace-comments` wasn't turned on, but I spotted it, may as well fix it.